contracts/EverlongStrategy.sol

```diff
@@ -572,9 +774,30 @@ contract EverlongStrategy is BaseStrategy {
     ///         a new position.
     /// @return True if a new position can be opened, false otherwise.
     function canOpenPosition() public view returns (bool) {
-        return
-            asset.balanceOf(address(this)) >
-            _poolConfig.minimumTransactionAmount;
+        uint256 currentBalance = asset.balanceOf(address(this));
+        return currentBalance > _poolConfig.minimumTransactionAmount;
```

Comment:
Something to think about is that the minimum transaction amount is in units of base rather than vault shares. If the token is wrapped or if `asBase = false`, the `currentBalance` will be in units of vaults shares rather than base.

Reply:
Great catch! Like the calcMaxLong issue should the fix for this be in a separate pr?

Reply:
Sounds good